bedrock/mozorg/models.py

```diff
@@ -129,3 +131,42 @@ def blog_title(self):
     @property
     def blog_link(self):
         return settings.BLOG_FEEDS[self.blog_slug]['url']
+
+    @classmethod
+    def parse_feed(cls, feed_id, feed_options):
```

Comment:
NIT: this `classmethod` doesn't actually use `cls` and could just be a regular function instead.